modular_zubbers/code/modules/cargo/packs/tech.dm

```diff
@@ -0,0 +1,14 @@
+/datum/supply_pack/science/secure_tech
+	name = "Secure Tech Replacement"
+	desc = "Lost your secure tech storage? Here's some replacements!"
+	cost = CARGO_CRATE_VALUE * 15 //3000
+	access = ACCESS_RD
```

Comment:
Should be command access, the same as the access requirements for the Secure Tech Storage door.